Makefile

```diff
@@ -285,21 +285,17 @@ endef
 ##@ Bundle
 
 .PHONY: operator-sdk
-OSDK = ./bin/operator-sdk
+OSDK = $(shell pwd)/bin/operator-sdk
```

Comment:
Why ./bin/... is not enough?

Reply:
Reverted, was done only to keep it similar to other sections.

Reply:
OK, unifying variables to use the same style is good, maybe in we can do this in a later commit.